### PR TITLE
Arf

### PR DESCRIPTION
**Pull request by mphprogrammer**

### CODE REVIEW

File: app/assets/javascripts/arf_gibills.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (plusjeff) [inline app/assets/javascripts/arf_gibills.js]: Can get removed

File: app/assets/stylesheets/arf_gibills.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the arf_gibills controller here.
```
Comment (plusjeff) [inline app/assets/stylesheets/arf_gibills.scss]: Can get removed

File: app/models/csv_files/csv_file.rb [hunk 3]
```diff
@@ -2,10 +2,18 @@ class CsvFile < ActiveRecord::Base
   attr_accessor :upload
 
   # Required for validation, and lists all allowed derived csv file tables.
-  STI = %W(WeamsCsvFile)
+  STI = { 
+    'WeamsCsvFile' => Weam,
```
Comment (plusjeff) [inline app/models/csv_files/csv_file.rb]: Can this be alphabetized?

**Comment by jeffmaher** [comment]
Haven't tested yet, but the code LGTM. Needs a merge up first.

**Pull request merged**